overridable/bool_or_string.go

```diff
@@ -25,6 +25,15 @@ func (bs *BoolOrString) Value(name string) interface{} {
 	return v
 }
 
+// ValueWithSuffix returns the value for the given repository and branch name.
+func (bs *BoolOrString) ValueWithSuffix(name, suffix string) interface{} {
+	v := bs.rules.MatchWithSuffix(name, suffix)
```

Comment:
![image](https://user-images.githubusercontent.com/19534377/107065642-49e35280-67dd-11eb-9058-d86d08007fbc.png)

Reply:
😆

Reply:
My first reaction: "Dude! I only added one method? What?"
Then: "Ahhhh, the `bs`. I get it. Okay. Nice. Nice. Going to save that pic for later use."